Applying improvements to the first half of the anglophone presentation

pedrohbraga opened the pull request:
Hi!

In this PR, I apply modifications to the first half of this workshop's anglophone presentation. The specific changes can be found in the descriptions of the commits associated with this pull request. 

In summary, they were:
- Fixing installation issues with `remotes`;
- Adding italics to Latin acronyms and words;
- Adding residuals segment lines to the scatterplot of simulated normal data;
- Improving the configuration of slide content;
- Deleting extra slides and fixing x-axis and y-axis labels
- Fixing x-axis labels;
- Correcting typos and adding words;
- Clarifying the language around the model prediction slides;
- Splitting the model prediction demonstration across several slides to clarify the explanation of where observations points would be expected to fall using the linear model on the Galumna data;
- Changing the little quiz question because it was very confusing and could have mixed interpretations;
- Changing omega for sigma in the slide notes;
- Adding presenter notes.

Modifications to the book and to the presentations in other languages will follow in the form of pull requests from separate branches. 

This part will be presented by me and I agree to its merging. However, the pull requests for the other parts should have presenter approval before being merged or only be merged following the workshop date.

hallegue commented:
I also included the translation from fr to en in this branch.

katherinehebert commented on pres-fr/workshop07-pres-fr.Rmd:
```diff
@@ -934,7 +933,9 @@ Vous devez vérifier que le modèle respecte toutes les suppositions de base:
 par(mar=c(4,4,.5,.5))
-plot(resid(M8) ~ fitted(M8), xlab = 'Valeurs pérdites', ylab = 'Résidus normalisés')
+plot(resid(M8) ~ fitted(M8), 
+     xlab = 'Valeurs pérdites', 
```
Please fix this small typo at line 937

katherinehebert commented on pres-en/workshop07-pres-en.Rmd:
```diff
@@ -1304,22 +1318,21 @@ Mixed models are very useful for taking into account the complex structure of ec
-Here the random effects are nested (i.e. Sites within forest) and not crossed
+Here the random effects are nested (i.e. Sites within forest) and not crossed.
+
+Why using `(1 | Foret / Site)` rather than `(1 | Foret) + (1 | Site)`? See [the answer in ![:faic](stack-exchange)](https://stats.stackexchange.com/questions/228800/crossed-vs-nested-random-effects-how-do-they-differ-and-how-are-they-specified)!
```
Please change to "Why use" and correct 1 | Foret to 1 | Forest

katherinehebert commented on pres-en/workshop07-pres-en.Rmd:
````diff
@@ -1050,23 +1032,52 @@ hist(resid(M8))
 ```
 
+---
+# Step 4: Interpretation and visualization
````
These slides are a great addition, thank you for this!

katherinehebert commented:
These changes are great, thank you! 👍 The branch will be ready to merge as soon as it is updated with the small corrections (just typos!) I have commented.

pedrohbraga committed:
Several fixes to the presentation

- Fixing installation issues with `remotes`;
- Adding italics to latin acronyms and words;
- Adding residuals segment lines to the scatterplot of simulated normal data;
- Improving the configuration of slide content;
- Deleting extra slide and fixing x-axis and y-axis labels

pedrohbraga committed:
Improving the clarity of some slides and concepts

- Fixing x-axis labels;
- Correcting typos and adding words;
- Clarifying the language around the model prediction slides;
- Splitting the model prediction demonstration across several slides to clarify the explanation of where observations points would be expected to fall using the linear model on the Galumna data;
- Changing the little quiz question because it was very confusing and could have mixed interpretation;
- Changing omega for sigma in the slide notes;
- Adding presenter notes

pedrohbraga committed:
Applies minor improvements to content
